build/assets.sh

```diff
@@ -15,22 +15,43 @@
 # limitations under the License.
 
 GIT_ROOT=$(dirname "${BASH_SOURCE}")/..
-YEAR=$(date +%Y)
 
-TMP_OUTPUT="/tmp/cadvisor_assets.go"
-ASSETS_OUTPUT=$GIT_ROOT/pages/static/assets.go
+ASSETS_INPUT_DIRS="$GIT_ROOT/pages/assets/js/... $GIT_ROOT/pages/assets/styles/..."
+ASSETS_OUTPUT_PATH="$GIT_ROOT/pages/static/assets.go"
+ASSETS_PACKAGE="static"
+
+TEMPLATES_INPUT_DIRS="$GIT_ROOT/pages/assets/html/..."
+TEMPLATES_OUTPUT_PATH="$GIT_ROOT/pages/templates.go"
+TEMPLATES_PACKAGE="pages"
 
 go get -u github.com/jteeuwen/go-bindata/...
 
-for f in $GIT_ROOT/pages/assets/**/*; do
-  if [ "$f" -nt $ASSETS_OUTPUT -o ! -e $ASSETS_OUTPUT ]; then
-    go-bindata -o $ASSETS_OUTPUT -pkg static $GIT_ROOT/pages/assets/...
-    cat build/boilerplate/boilerplate.go.txt | sed "s/YEAR/$YEAR/" > "${TMP_OUTPUT}"
-    echo -e "// generated by build/assets.sh; DO NOT EDIT\n" >> "${TMP_OUTPUT}"
-    cat "${ASSETS_OUTPUT}" >> "${TMP_OUTPUT}"
-    gofmt -w -s "${TMP_OUTPUT}"
-    mv "${TMP_OUTPUT}" "${ASSETS_OUTPUT}"
-    break
+build_asset () {
+  local package=$1
+  local output_path=$2
+  local input_dirs=${@:3}
+  local tmp_output="/tmp/cadvisor_asset_build.go"
```

Comment:
Ah, my bad. I thought "mktemp" was an arbitrary name, didn't realize it was a utility.